seeds/userSeeds.json

```diff
@@ -1,15 +1,42 @@
 [
-    {
-      "username": "JamesMurray",
-      "password": "password12345"
-    },
-    {
-      "username": "Salaster",
-      "password": "password12345"
-    },
-    {
-      "username": "JoeGatto",
-      "password": "password12345"
-    }
+  {
+    "username": "JamesMurray",
+    "password": "password12345"
+  },
+  {
+    "username": "Sal Vulacano",
+    "password": "password12345"
+  },
+  {
+    "username": "Joe Gatto",
+    "password": "password12345"
+  },
+  {
+    "username": "Theo Von",
+    "password": "password12345"
+  },
+  {
+    "username": "Micheal Carbanaro",
+    "password": "password12345"
+  },
+  {
+    "username": "JoeGatto",
```

Comment:
You've got a few of the same username listed here, which is going to cause a problem with the current model validation that each username must be unique in /models/User.js